.github/workflows/go.yml

```diff
@@ -365,6 +365,9 @@ jobs:
           # make the cli.js executable
           chmod u+x ./bin/cli.js
 
+          # copy the README
+          cp ${{ github.workspace }}/README.md ./README.md
```

Comment:
I think we should have an NPM specific README because the main README just tells you how to install it which is not a problem NPM users will have.